tests/unit/adapters/osf-adapter-test.js

```diff
@@ -399,3 +482,44 @@ test('#_handleRelatedRequest checks if relationship supports bulk', function(ass
         );
     }
 });
+
+test('#updateRecord handles both dirtyRelationships and the parent record', function(assert) {
+    let adapter = this.subject();
+
+    this.inject.service('store');
+    let store = this.store;
+
+    let node = FactoryGuy.make('node');
+    Ember.run(() => node.set('title', 'The meaning of life'));
```

Comment:
I'm not especially keen on attempting to check that a model gets 'saved' in a unit test. Stubbing the super call (L502) adequate here.